pkg/nsx/services/common/types.go

```diff
@@ -63,6 +63,8 @@ const (
 	TagValueGroupSrc                string = "source"
 	TagValueGroupDst                string = "destination"
 	TagValueVersion10               string = "1.0"
+	TagScopeVMSubnetNamespaceUID    string = "nsx-op/vm_subnet_namespace_uid"
```

Comment:
I suggest change TagScopeVMSubnetNamespaceUID to TagScopeVMNamespaceUID:"nsx-op/vm_namespace_uid"
TagScopeVMSubnetNamespace to TagScopeVMNamespace:"nsx-op/vm_namespace"
since both tagas will be used for VM subnet and subnetport.

Reply:
ok